packages/gatsby-plugin-image/src/components/layout-wrapper.tsx

```diff
@@ -46,30 +46,28 @@ export function getSizer(
 ): string {
   let sizer: string | null = null
   if (layout === `fullWidth`) {
-    sizer = `<div aria-hidden="true" style="padding-top: ${
+    sizer = `<div data-gatsby-image-sizer aria-hidden="true" style="padding-top: ${
```

Comment:
data-gatsby-image-sizer is something leftover from an experiment :)

```suggestion
    sizer = `<div aria-hidden="true" style="padding-top: ${
```